lib/plugin.js

```diff
@@ -134,7 +134,7 @@ const setupProgress = (axios) => {
     if (config && config.progress === false) {
       return
     }
-
+    $loading().start()
```

Comment:
Would you please also explain why this is required? Preferably adding a comment on top of it would be awesome!

Reply:
@pi0 
We need to start the progress bar in onRequest. If the request response header contains content-length, we can rely on the $loading().set in onProgress to automatically start the progress bar, but if the response header does not contain content-length, then there will be a problem.